bundles/org.openhab.binding.verisure/README.md

```diff
@@ -31,11 +30,14 @@ This binding supports the following thing types:
 
 ## Binding Configuration
 
-You will have to configure the bridge with username and password, these must be the same credentials as used when logging into https://mypages.verisure.com.
+You will have to configure the bridge with username and password of a pre-defined user on https://mypages.verisure.com that has not activated Multi Factor Authentication (MFA/2FA). 
```

Comment:
Please use a link like for example: [Verisure page](https://mypages.verisure.com)

Reply:
Fixed!